src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -390,7 +392,18 @@ public Command getDriveToPoseCommand() {
   }
 
   /** Runs the drive at the desired speeds set in (@Link setGoalSpeeds) */
-  public void runVelocity() {
+  public void runVelocity(Rotation2d desiredRotation) {
+    if (isAngleAlign) {
+      ProfiledPIDController angleController =
```

Comment:
This should probably be a class variable rather than a local variable. It is resetting every frame which is probably not helping your cause